src/main/java/com/easypost/Constants.java

```diff
@@ -58,7 +58,7 @@ public abstract static class ErrorCodes {
 
     public abstract static class CarrierAccountTypes {
         public static final List<String> CARRIER_TYPES_WITH_CUSTOM_WORKFLOW = ImmutableList.of("FedexAccount",
-                "UpsAccount");
+                "UpsAccount", "FedexSmartpostAccount");
```

Comment:
question: can these be per line or nah?

Reply:
Probably, but I don't feel like fighting with the Java linter right now 🙃